x-pack/test/security_solution_endpoint/configs/config.base.ts

```diff
@@ -55,6 +56,7 @@ export const generateConfig = async ({
 
   return {
     ...baseConfig.getAll(),
+    testConfigCategory: ScoutTestRunConfigCategory.API_TEST,
```

Comment:
This config is for the FTR UI tests, not the API.

Reply:
Good spot, that was a copy/paste error.
Fixed in 31643cf